### DISCUSSION TITLE
error getting credentials

### DISCUSSION THREAD

**Discussion by Luxvao** [rancher-sandbox/rancher-desktop#2867] [Q&A] [answered]
I tried to build a Dockerfile with containerd and nerdctl and I got this error: 
`error: failed to solve: error getting credentials - err: exit status 22, out: ```
Then I tried to run nerdctl login and I got this error:
`error saving credentials: error storing credentials - err: exit status 22, out: ``` (I used my docker hub username and passwd)

**Comment by adamkpickering** [comment]
Hi, this sounds like a familiar issue. Can you please look through the issues, and if you don't find yours, create a new one? As it is, it is difficult to solve your problem as I don't have enough information about your setup :)

**Comment by Luxvao** [comment, marked as answer]
Hi it actually started working when I restarted rancher desktop. I have no idea why and I don't need to. All I know is that is works. (I forgot to close this question). Thanks anyway :)
